src/ts/util/ResourceManager.ts

```diff
@@ -0,0 +1,36 @@
+module util.resource {
+    var _resources : {[type : string] : {[key : string] : any}} = {};
```

Comment:
Since this contains state it should be a shared object instead of a static value.